specs/phase0/fork-choice.md

```diff
@@ -76,11 +86,16 @@ Any of the above handlers that trigger an unhandled exception (e.g. a failed ass
 
 ### Configuration
 
-| Name                   | Value        |
-| ---------------------- | ------------ |
-| `PROPOSER_SCORE_BOOST` | `uint64(40)` |
+| Name                                  | Value        |
+| ------------------------------------- | ------------ |
+| `PROPOSER_SCORE_BOOST`                | `uint64(40)` |
+| `REORG_WEIGHT_THRESHOLD`              | `uint64(20)` |
+| `REORG_PARENT_WEIGHT_THRESHOLD`       | `uint64(160)`|
```

Comment:
@michaelsproul could you clarify why `REORG_PARENT_WEIGHT_THRESHOLD` is `160`?

Reply:
The calculation to derive it is here: https://ethresear.ch/t/selfish-mining-in-pos/15551/5

Do you want to link to that, or should we include a short summary of that idea?

Reply:
The intuitive explanation is that we need 80% of attesters from both slot N - 1 and N to see N - 1 as the head.

Reply:
Updated the PR description with the link